*** Makefile	2014-06-06 19:36:23.665955385 +0900
--- Makefile.patched	2014-06-06 19:35:09.531882188 +0900
***************
*** 1,4 ****
! CFLAGS= -I. -Wall
  SRCS=	bin2hex.c endian.c aout16.c aout32.c bin.c com.c elf.c exe.c rel.c sav.c
  OBJS=	bin2hex.o endian.o aout16.o aout32.o bin.o com.o elf.o exe.o rel.o sav.o
  
--- 1,4 ----
! CFLAGS= -Wall
  SRCS=	bin2hex.c endian.c aout16.c aout32.c bin.c com.c elf.c exe.c rel.c sav.c
  OBJS=	bin2hex.o endian.o aout16.o aout32.o bin.o com.o elf.o exe.o rel.o sav.o
  
